rust/ql/test/extractor-tests/macro_expansion/test.ql

```diff
@@ -0,0 +1,6 @@
+import rust
+import TestUtils
```

Comment:
Perhaps some "printAst"-like output would be helpful to show the expanded AST?

Reply:
I thought about that, but I find printAst-heavy tests to tend to need a lot of updating. But we can have it in if you prefer.

Reply:
added a PrintAst test